src/cascadia/UnitTests_Control/ControlInteractivityTests.cpp

```diff
@@ -0,0 +1,341 @@
+                                  til::point{ 0, 0 },
+                                  { false, false, false });
+
+        Log::Comment(L"Scroll up 19 more times, to the top");
+        for (int i = 0; i < 20; ++i)
+        {
+            expectedTop--;
+            interactivity->MouseWheel(modifiers,
+                                      WHEEL_DELTA,
+                                      til::point{ 0, 0 },
+                                      { false, false, false });
+        }
+        Log::Comment(L"Scrolling up more should do nothing");
+        expectedTop = 0;
+        interactivity->MouseWheel(modifiers,
+                                  WHEEL_DELTA,
+                                  til::point{ 0, 0 },
+                                  { false, false, false });
+        interactivity->MouseWheel(modifiers,
+                                  WHEEL_DELTA,
+                                  til::point{ 0, 0 },
+                                  { false, false, false });
+
+        Log::Comment(L"Scroll down 21 more times, to the bottom");
+        for (int i = 0; i < 21; ++i)
+        {
+            expectedTop++;
+            interactivity->MouseWheel(modifiers,
+                                      -WHEEL_DELTA,
+                                      til::point{ 0, 0 },
+                                      { false, false, false });
+        }
+        Log::Comment(L"Scrolling up more should do nothing");
+        expectedTop = 21;
+        interactivity->MouseWheel(modifiers,
+                                  -WHEEL_DELTA,
+                                  til::point{ 0, 0 },
+                                  { false, false, false });
+        interactivity->MouseWheel(modifiers,
+                                  -WHEEL_DELTA,
+                                  til::point{ 0, 0 },
+                                  { false, false, false });
+    }
+
+    void ControlInteractivityTests::CreateSubsequentSelectionWithDragging()
+    {
+        // This is a test for GH#9725
+        WEX::TestExecution::DisableVerifyExceptions disableVerifyExceptions{};
+
+        auto [settings, conn] = _createSettingsAndConnection();
+        auto [core, interactivity] = _createCoreAndInteractivity(*settings, *conn);
+        _standardInit(core, interactivity);
+
+        // For this test, don't use any modifiers
+        const auto modifiers = ControlKeyStates();
+        const TerminalInput::MouseButtonState leftMouseDown{ true, false, false };
+        const TerminalInput::MouseButtonState noMouseDown{ false, false, false };
+
+        const til::size fontSize{ 9, 19 };
+
+        Log::Comment(L"Click on the terminal");
+        const til::point terminalPosition0{ 0, 0 };
+        const til::point cursorPosition0 = terminalPosition0 * fontSize;
+        interactivity->PointerPressed(cursorPosition0,
+                                      leftMouseDown,
+                                      WM_LBUTTONDOWN, //pointerUpdateKind

```

Comment:
don't love the narrowing through old win32 message params lol

Reply:
Yea it's not the best, but like, we kinda already were. Figured it was more sensible than: 
* using the `Windows.UI` enum
* rolling my own enum for this